### PR TITLE
auto merge updates to https://github.com/bitnami/charts

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

**Comment by prydonius** [comment]
@sameersbn this is pending authentication right?

**Comment by sameersbn** [comment]
@prydonius no, the authentication for github.com is already performed in previous steps. We should be good to merge this PR as-is.

File: circle/docker-release-image.sh [hunk 1]
```diff
@@ -412,6 +412,14 @@ if [[ -n $CHART_REPO ]]; then
           error "Could not create pull request"
           exit 1
         fi
+
+        # auto merge updates to https://github.com/bitnami/charts
+        if [[ $CHART_REPO == https://github.com/bitnami/charts ]]; then
```
Comment (prydonius) [inline circle/docker-release-image.sh]: should https://github.com/bitnami/charts be quoted?

Comment (sameersbn) [inline circle/docker-release-image.sh]: its not required with `[[` but i'll add it anyway ¯\_(ツ)_/¯

**Pull request merged**